lib.urp

```diff
@@ -3,6 +3,7 @@ library $/bootstrap/bootstrap5
 library $/ajaxUpload
 library ckeditor
 library chartist
```

Comment:
It probably isn't worth leaving multiple chart libraries in this library.  How would you feel about removing Chartist?

Reply:
Happy to do it.